lms/static/js/demographics_collection/DemographicsCollectionModal.jsx

```diff
@@ -436,7 +458,7 @@ class DemographicsCollectionModal extends React.Component {
           </Wizard.Page>
           <Wizard.Closer>
             <div className="demographics-modal-closer m-sm-0">
-            <i class="fa fa-check" aria-hidden="true"></i>
+              <i class="fa fa-check" aria-hidden="true"></i>
```

Comment:
Should this be `className` over `class`?

Reply:
Ohh nice catch!

Reply:
🛠️ 'd